VideoPlayer.js

```diff
@@ -1061,6 +1078,18 @@ export default class VideoPlayer extends Component {
         );
     }
 
+    /**
+     * Render the seekbar and attach its handlers
```

Comment:
Be careful about the description, it's not the seekbar

Reply:
you're right thx for the feedback 👍